website/docs/r/sentinel_policy.html.markdown

```diff
@@ -0,0 +1,44 @@
+---
+layout: "tfe"
+page_title: "Terraform Enterprise: tfe_sentinel_policy"
+sidebar_current: "docs-tfe-sentinel-policy"
+description: |-
+Sentinel Policy as Code is an embedded policy as code framework integrated with Terraform Enterprise.
+---
+
+# tfe_sentinel_policy
+
+Sentinel Policy as Code is an embedded policy as code framework integrated
+with Terraform Enterprise.
+
+Policies are configured on a per-organization level, and are enforced on
+all of an organization's workspaces during runs. Each plan's changes are
+validated against the policy prior to the apply step.
+
+## Example Usage
+
+Basic usage:
+
+```hcl
+resource "tfe_sentinel_policy" "policy" {
+	name = "my-policy-name"
+	organization = "my-org-name"
+	policy = "main = rule { true }"
+	enforce_mode = "hard-mandatory"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) Name of the policy.
+* `organization` - (Required) Name of the organization.
+* `policy` - (Required) The actual policy itself.
+* `enforce_mode` - (Required) The enforcement level of the policy. Valid
+  values are `advisory`, `hard-mandatory` and `soft-mandatory`. Defaults
+  to `soft-mandatory`.
```

Comment:
is this defaulted in the API?

Reply:
Yes... But will update the resource as well... Fixed in next commit.